public/app/features/dashboard/dashgrid/DashboardPanel.tsx

```diff
@@ -14,6 +14,8 @@ import { PanelEditor } from './PanelEditor';
 import { PanelModel } from '../panel_model';
 import { DashboardModel } from '../dashboard_model';
 import { PanelPlugin } from 'app/types';
+import { ResizableBox } from 'react-resizable';
```

Comment:
Note that we depend on react-resizable here since its included in react-draggable. Maybe we should add react-resizable to our package.json as well?